ansible/configs/kni-osp/files/install-config.yaml.j2

```diff
@@ -0,0 +1,79 @@
+apiVersion: v1
+baseDomain: {{ osp_cluster_dns_zone }}
+metadata:
+  name: {{ guid }}
+networking:
+  networkType: {{ my_ocp_networking }}
+  machineCIDR: 10.20.0.0/24
+compute:
+- name: worker
+  replicas: 2
+controlPlane:
+  name: master
+  replicas: 3
+  platform:
+    baremetal: {}
+platform:
+  baremetal:
+    provisioningNetworkCIDR: 172.22.0.0/24
+    provisioningNetworkInterface: ens3
+    apiVIP: 10.20.0.110
+    ingressVIP: 10.20.0.112
+    dnsVIP: 10.20.0.111
+{% if my_ocp_offline %}
+    bootstrapOSImage: http://10.20.0.5/images/RHCOS_QEMU_IMAGE
+    clusterOSImage: http://10.20.0.5/images/RHCOS_OPENSTACK_IMAGE
+{% endif %}
+    hosts:
+      - name: master-0
+        role: master
+        bmc:
+          address: ipmi://10.20.0.3:master-0BMC
+          username: admin
+          password: redhat
+        bootMACAddress: de:ad:be:ef:00:40
+        hardwareProfile: default
+      - name: master-1
+        role: master
+        bmc:
+          address: ipmi://10.20.0.3:master-1BMC
+          username: admin
+          password: redhat
+        bootMACAddress: de:ad:be:ef:00:41
+        hardwareProfile: default
+      - name: master-2
+        role: master
+        bmc:
+          address: ipmi://10.20.0.3:master-2BMC
+          username: admin
+          password: redhat
+        bootMACAddress: de:ad:be:ef:00:42
+        hardwareProfile: default
+      - name: worker-0
+        role: worker
+        bmc:
+          address: ipmi://10.20.0.3:worker-0BMC
+          username: admin
+          password: redhat
+        bootMACAddress: de:ad:be:ef:00:50
+        hardwareProfile: unknown
+      - name: worker-1
+        role: worker
+        bmc:
+          address: ipmi://10.20.0.3:worker-1BMC
+          username: admin
+          password: redhat
+        bootMACAddress: de:ad:be:ef:00:51
+        hardwareProfile: unknown
+
+sshKey: 'SSH_KEY'
```

Comment:
This should be set to a var and defined.

Reply:
this is to be picked by one of the scripts during the initial iteration of the lab ... a more automated version will follow that will bring a pre-installed CNV environment. This iteration is about getting students to set it up